docs/introduction/Sidechain-SDK-extension.rst

```diff
@@ -306,22 +306,22 @@ ApplicationState:
   Try<ApplicationState> onRollback(byte[] version);
   }
 
-For example, the custom application may have the possibility to tokenize cars by the creation of Box entries - let us call them CarBox. Each CarBox token should represent a unique car by having a unique *VIN* (Vehicle Identification Number). To do this, Sidechain developer may define ApplicationState to store the list of actual VINs and reject transactions with CarBox tokens with VIN already existing.
+For example, the custom application may have the possibility to tokenize cars by the creation of Box entries - let us call them CarBox. Each CarBox token should represent a unique car by having a unique *VIN* (Vehicle Identification Number). To do this, the sidechain developer may define ApplicationState to store the list of actual VINs and reject transactions with CarBox tokens with VIN already existing.
 
 The next custom state checks could be done here:
 
-  * ``public boolean validate(SidechainStateReader stateReader, SidechainBlock block)`` --  any custom block validation could be done here. If the function returns false, then the block will not be accepted by the Sidechain Node.
+  * ``public boolean validate(SidechainStateReader stateReader, SidechainBlock block)`` -- any custom block validation could be done here. If the function returns false, then the block will not be accepted by the sidechain node.
   
-  * ``public boolean validate(SidechainStateReader stateReader, BoxTransaction<Proposition, Box<Proposition>> transaction)`` -- any custom checks for the transaction could be done here if the function returns false then transaction is assumed as invalid and for example will not be included in a memory pool. 
+  * ``public boolean validate(SidechainStateReader stateReader, BoxTransaction<Proposition, Box<Proposition>> transaction)`` -- any custom checks for the transaction could be done here. If the function returns false, then the transaction is assumed as invalid and, for example, will not be included in a memory pool. 
 
   * ``public Try<ApplicationState> onApplyChanges(SidechainStateReader stateReader, byte[] version, List<Box<Proposition>> newBoxes, List<byte[]> boxIdsToRemove)`` -- any specific action after block applying in State could be defined here.
   
-  * ``public Try<ApplicationState> onRollback(byte[] version)`` -- any specific action after rollback of State (for example in case of fork/invalid block) could be defined here
+  * ``public Try<ApplicationState> onRollback(byte[] version)`` -- any specific action after rollback of State (for example, in case of fork/invalid block) could be defined here.
```

Comment:
@zennermac 

  * ``public Try<ApplicationState> onRollback(byte[] version)`` -- any specific action after a rollback of the state (for example, in case of fork/invalid block) could be defined here.